<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshExplorationWidget</class>
 <widget class="QWidget" name="MeshExplorationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>1003</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="meshNameFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Slice type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sliceTypeComboBox">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Type of slicing&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;	Extract cells 	&lt;/span&gt;shows whole mesh elements without cutting through them (for analysis of the mesh)&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;	Clip &lt;/span&gt;			cuts through the cells (for viewing the computed solution)&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <item>
        <property name="text">
         <string>Extract cells</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Clip</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="clippingPlanesTableWidget">
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Enabled</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Invert</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="autoUpdateCheckBox">
       <property name="toolTip">
        <string>Update the view automatically when the cutting planes change</string>
       </property>
       <property name="text">
        <string>Update automatically</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="updateButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Manually update the views</string>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/VesselMeshing.qrc">
         <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sliceTypeComboBox</tabstop>
  <tabstop>clippingPlanesTableWidget</tabstop>
  <tabstop>autoUpdateCheckBox</tabstop>
  <tabstop>updateButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/VesselMeshing.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>autoUpdateCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>updateButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>956</y>
    </hint>
    <hint type="destinationlabel">
     <x>749</x>
     <y>956</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
